vspec/vspec2x.py

```diff
@@ -22,6 +22,8 @@
 import sys
 import vspec
 
+VERSION = '5.0-dev'
```

Comment:
Consider putting the definition into a file that can be included or read in here as well as included automatically in documentation or release notes?

Reply:
We could have it in a separate file like in the VSS repo, but that would require some extra work.
Possibly integrate it in some way with what we already have in https://github.com/COVESA/vss-tools/blob/master/setup.py. But that would require some extra work and verification to make sure it works as expected in all deployment alternatives with have (like venv and pypi)